doc/api/net.md

```diff
@@ -40,8 +40,9 @@ added: v0.1.90
 
 * {Error}
 
-Emitted when an error occurs.  The [`'close'`][] event will be called directly
-following this event.  See example in discussion of `server.listen`.
+Emitted when an error occurs. Unlike [`net.Socket`][], the [`'close'`][]
+event will NOT be called directly following this event, unless you call
+`server.close()` manually.  See example in discussion of `server.listen`.
```

Comment:
Please avoid the use of pronouns like `you` in the documentation. Also, please use either italics or bold for emphasis as opposed to upper case (e.g. `NOT`) This can be reworded as:

```
Emitted when an error occurs., Unlike [`net.Socket'`[], the [`'close'`][]
event will **not** be called directly following this event unless
`server.close()` is manually called. See the example in the discussion
of `server.listen()`.
```